books/RayTracingTheNextWeek.html

```diff
@@ -656,40 +660,67 @@
 
 </div>
 
-
-An Optimized AABB Hit Method
------------------------------
-In reviewing this intersection method, Andrew Kensler at Pixar tried some experiments and proposed
-the following version of the code. It works extremely well on many compilers, and I have adopted it
-as my go-to method:
+<div class='together'>
+The new code above introduces new interval functions we need to write: `interval::is_empty()`,
```

Comment:
Consider rephrasing this as "The new code above relies on three new `interval` methods that we haven't defined: ...`.

"relies on" is a little more clear compared to "introduces". I've seen the book use both "method" and "function" interchangeably, which is fine, though `method` makes it a bit clearer that you're talking about instance functions.

Reply:
Rephrased. Note that two of the new functions are class functions, and one is a standalone function.

Reply:
Ah, you're right. I missed that `span()` is a standalone function. In that case I would just rephrase this like so:

    The new code above relies on three new functions that we haven't defined: `interval::is_empty()`, `interval::intersect()`, and `span()`.